javascript/react/src/components/ArqueroDetailsList/__tests__/Columns/Columns.tsx

```diff
@@ -0,0 +1,41 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+import type { ArqueroDetailsListProps } from '@datashaper/react'
+import { ArqueroDetailsList } from '@datashaper/react'
+import { Checkbox } from '@fluentui/react'
+import { memo } from 'react'
+import styled from 'styled-components'
+
+import { Table } from '../ArqueroDetailsList.styles.js'
+import { useCheckboxConfigs, useColumns } from './Columns.hooks.js'
+
+export const Columns: React.FC<ArqueroDetailsListProps> = memo(
+	function Columns({ table, ...args }) {
+		const { checkboxes } = useCheckboxConfigs(table)
+
+		const { columns } = useColumns(checkboxes)
+
+		if (!table) {
+			return <div>Loading...</div>
```

Comment:
Hmm, this is fine for now but I wonder if we should think about including shimmer components for when things are in a loading-state.

Reply:
It's just a story so I'm not super concerned